lib/tasks/approval_cleanup.rake

```diff
@@ -0,0 +1,29 @@
+require 'rake'
+
+namespace :approval do
+  desc "Resource cleanup tasks"
+  namespace :workflows do
+    desc "Cleanup workflows with empty associated group references"
+    task :cleanup => :environment do
+      workflows = Workflow.where(:group_refs => []).where.not(:name => "Always approve")
+      puts "The follow workflows with empty group references will be deleted:  #{workflows.pluck(:id, :name)}"
+      workflows.destroy_all
+    end
+  end
+
+  namespace :requests do
+    desc "Cleanup requests older than specified days"
+    task :cleanup, [:days] => [:environment] do |_t, args|
+      days = Integer(args.days)
+
+      if days.zero?
+        puts "Not allowed to delete all requests in this taks"
+        exit
+      end
+
+      requests = Request.where("created_at < ?", Time.zone.today - days)
+      puts "Requests #{requests.pluck(:id)} created #{days} days ago will be cleaned up"
```

Comment:
This wording is incorrect, the requests were **not all** created `#` days ago, it is anything older than `#` days.  And listing the IDs at the end will be easier to read:

Suggest:
"Requests older than #{days} days will be deleted: #{requests.pluck(:id)}"